apps/api/src/app/workflows-v2/workflow.controller.e2e.ts

```diff
@@ -223,6 +223,20 @@ describe('Workflow Controller E2E API Testing', () => {
           }
         }
       });
+      it('should show control value required when empty', async () => {
```

Comment:
😄
```suggestion

      it('should show control value required when empty', async () => {
```

Reply:
whats the change? It looks the same @ChmaraX

Reply:
newline between functions